onnxruntime/core/mlas/inc/mlas_qnbit.h

```diff
@@ -77,3 +77,134 @@
     size_t BlkBitWidth,
     size_t BlkLen
 );
+
+/**
+ * @brief Define compute types of block quantization
+ */
+typedef enum {
+    CompUndef = 0, /*!< undef */
+    CompFp32 = 1,  /*!< input fp32, accumulator fp32 */
+    CompFp16 = 2,  /*!< input fp16, accumulator fp16 */
+    CompBf16 = 3,  /*!< input bf16, accumulator fp32 */
+    CompInt8 = 4   /*!< input int8, accumulator int32 */
+} MLAS_COMPUTE_TYPE;
+
+/**
+ * @brief Data parameters for Q4 GEMM routine
+ *        C = A * B + Bias
+ *        A must be a float32 matrix
+ *        B must be a quantized and packed int4 blob
+ *        All except C are [in] parameters
+ */
+struct MLAS_NBITS_GEMM_DATA_SIMPLE_PARAMS {
```

Comment:
nit: name it as MLAS_NBITS_GEMM_DATA_PACKED_PARAMS

Reply:
renamed